components/bl-backdrop-component/.gitignore

```diff
@@ -1 +1 @@
-/dist/
\ No newline at end of file
+/dist/!/.editorconfig
```

Comment:
editor config could be applied globally for your ide, don't add it to gitignore.